libcalico-go/lib/backend/k8s/resources/node_test.go

```diff
@@ -77,6 +77,84 @@ var _ = Describe("Test Node conversion", func() {
 		Expect(asn.String()).To(Equal("2546"))
 	})
 
+	It("should ignore RR cluster ID if its an invalid IPv4 address", func() {
+		l := map[string]string{"net.beta.kubernetes.io/role": "master"}
```

Comment:
what is this label? is it needed for this test?

Reply:
At a minimum this should be "control-plane" instead of master, but I don't actually think we need any labels for this test so could just be removed.